blueocean-dashboard/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "devDependencies": {
     "@jenkins-cd/eslint-config-jenkins": "0.0.2",
-    "@jenkins-cd/js-builder": "0.0.47",
+    "@jenkins-cd/js-builder": "0.0.49-beta6",
```

Comment:
That is fixing a bug that we discovered when I moved the NotFound to core and had to introduce `react-router` as dep. Without that version that leads to an exception for the bundle. Kudos to @tfennelly